pkg/digest/digest.go

```diff
@@ -34,7 +36,9 @@ func Sha256(value string) string {
 func Sha1(contents []string) string {
 	h := sha1.New()
 	for _, content := range contents {
-		io.WriteString(h, content)
+		if _, err := io.WriteString(h, content); err != nil {
+			fmt.Fprintf(os.Stderr, "fail to write string to io.Write %v", err)
```

Comment:
failed to write string to io.Write: %v

please use `failed` and `:`

Reply:
ok, I add them.